cadc-file-validate/README.md

```diff
@@ -1,56 +1,56 @@
 # cadc-file-validate 0.1
 
-Command line tool to ensure validity of the information stored in the Inventory Database and what is served from a Storage Adapter.  
+Process to ensure validity of the information stored in the Inventory Database and what is served from a Storage Adapter.  
 
 This is intended to run periodically at any site to determine what changes, if any, are to be made to either the Storage Inventory database, or
 the storage system.
 
 ## Configuration
 
 A file called `cadc-file-validate.properties` should be available from the `/config` directory.  All properties in the file are loaded and set
-as System properties:
-
-```java
-Properties properties = new Properties();
-properties.load("/config/cadc-file-validate.properties");
-System.setProperties(properties);
-```
+as System properties.
 
 ### Sample configuration (`cadc-file-validate.properties`)
 
 ```
+#######
+## Common settings
+#######
+org.opencadc.inventory.logging = INFO
```

Comment:
I was thinking this would have to be something like:
```
org.opencadc.{name}.logLevel = INFO
org.opencadc.{name}.logPackages = org.opencadc.inventory ca.nrc.cadc.db ... 
```
in order to be usable to read system props and execute Log4jInit.setLevel(...) calls for an arbitrary set of packages.  Or maybe something like this would allow more flexibility:
```
org.opencadc.{name}.logging.info = org.opencadc.inventory ca.nrc.cadc.db ...
org.opencadc.{name}.logging.debug = org.opencadc.{name} ca.nrc.cadc.net ...
```
The approach has to be usable for the other 3 tools we develop as well and because they bring networking into play there will be more packages to want to fiddle log levels.

Reply:
I see.  Would this be a little more complicated than what is needed?  Would the use case be, "put `cadc-file-validate` in `DEBUG` mode"?  If so, then we can simply say that all `org.opencadc.inventory.*`, `ca.nrc.cadc.net`, `ca.nrc.cadc.util`, `ca.nrc.cadc.db` packages are automatically put into `DEBUG` level.

If that is going to be too noisy, or if we need a finer grained configuration, then maybe we could use what `log4j` does and start with the `logging` keyword to easily find them:

```
logging.org.opencadc.{name} = INFO
logging.org.opencadc.inventory = DEBUG
logging.ca.nrc.cadc.net = TRACE
```